ochami-cli

```diff
@@ -230,7 +434,8 @@ def is_access_token_valid(token: str) -> bool:
 def setArgs():
     parser = argparse.ArgumentParser()
     parser.add_argument("--access-token", dest="access_token", default="", type=str)
-    parser.add_argument("--access-token-file", dest="access_token_file", default=".ochami-token", type=str)
+    parser.add_argument("--access-token-file", dest="access_token_file", default=".access-token", type=str)
+    parser.add_argument("--ca-cert", dest="ca_cert", type=str, default="", help="set path for CA certificate")
```

Comment:
does this also read from system store w/ requests?

Reply:
According to [this](https://stackoverflow.com/questions/42982143/python-requests-how-to-use-system-ca-certificates-debian-ubuntu#42982144), `requests` reads from a Python-specific store. In order to read the system store, you need (e.g. on a Rocky system):

```
export REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-bundle.crt
```